napari/_app_model/actions/_help_actions.py

```diff
@@ -0,0 +1,67 @@
+"""Actions related to the 'Help' menu that do not require Qt.
+
+View actions that do require Qt should go in
+`napari/_qt/_qapp_model/qactions/_help.py`.
+"""
+import webbrowser
+from typing import List
+
+from app_model.types import Action
+from packaging.version import parse
+
+from napari import __version__
+from napari._app_model.constants import CommandId, MenuId
+
+v = parse(__version__)
+VERSION = "dev" if v.is_devrelease else str(v)
+
+
+HELP_ACTIONS: List[Action] = [
+    Action(
+        id=CommandId.NAPARI_GETTING_STARTED,
+        title=CommandId.NAPARI_GETTING_STARTED.title,
+        callback=lambda: webbrowser.open(
+            f'https://napari.org/{VERSION}/tutorials/start_index.html'
```

Comment:
@psobolewskiPhD how about we move all these URLS to a constant dictionary next to `VERSION` that way we can write a simple test that checks if the urls keep working moving forward.

We can use requests to write this test.

Something like:


```python
HELP_URLS = {
    "getting-started":, f'https://napari.org/{VERSION}/tutorials/start_index.html',
     "tutorials": f'https://napari.org/{VERSION}/tutorials/index.html',
    # ...
}
```

And the test could be something like

```python

import requests  # See https://requests.readthedocs.io/en/latest/

from napari._app_model.actions._help_actions import HELP_URLS


def test_help_urls():
    for key, url in HELP_URLS.items():
        r = requests.head(url)
        r.raise_for_status()
```

Reply:
Thanks a ton @goanpeca 
Really helpful!

Implemented the test slightly differently:
I wanted the test to return info about each link, because I knew `release_notes` doesn't work for `dev` (aka CI).
So I read up on parametrize from pytest and used that to check each url in the dict, skipping the release notes one.